tb_arch.py

```diff
@@ -0,0 +1,242 @@
+import turtle
```

Comment:
please add appropriate comments of the code what it is doing

Reply:
created a new pull request for this.